### DISCUSSION TITLE
Copilot not suggesting anything when Visual Studio is started with admin rights

### DISCUSSION THREAD

**Discussion by Xteuk** [community/community#47638] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

As said in the title: If my Visual Studio (Entreprise, 2022) is started with admin rights*, it does not suggest anything on my code (C#).
Though implicit, let's precise that it works just fine when started in normal mode.
The output seems fine, but sticks to those two lines:
  GitHub Copilot 1.65.0.0 (v1.65.0.0@94a690b91)
  GitHub Copilot Agent v1.65.0

* sorry I'm not english, so this might not be the exact terms. Maybe "advanced permissions"?
* Also, I need that to be allowed to debug services

**Comment by Xteuk** [comment, marked as answer]
Maybe due to an update of copilot or VS ? But it works, now.

**Reply by mdhafizskkhan** [reply]
My output hota nahuhe

**Comment by mdhafizskkhan** [comment]
Kese output ko start kare
